fcl/reco/Stage1/Run1/stage1_multiTPC_icarus_gauss.fcl

```diff
@@ -39,24 +39,14 @@ services.message.destinations :
   STDCOUT:
   {
      type:      "cout"      #tells the message service to output this destination to cout
-     threshold: "DEBUG"     #tells the message service that this destination applies to WARNING and higher level messages
+     threshold: "WARNING"     #tells the message service that this destination applies to WARNING and higher level messages
      categories:
      {
        Cluster3D:
        {
          limit: -1
          reportEvery: 1
        }
-       PMAlgTracker:
-       {
-         limit: -1
-         reportEvery: 1
-       }
-       default:
-       {
-         limit: 0  #don't print anything at the infomsg level except the explicitly named categories
-         reportEvery: 0
-       }
```

Comment:
What happens when there is no `default` section?

Reply:
It is defined in the common fcl file...